README.md

```diff
@@ -320,10 +348,11 @@ new Vue({
 </template>
 
 <script>
-import { AeMain, aeHelperMixin } from '@aeternity/aepp-components'
+import { AeMain, mixins } from '@aeternity/aepp-components'
 
 export default {
   name: 'app',
+  mixins: [mixins.events], 
```

Comment:
Is tree shaking works in this case? Is unused mixins will be removed from the bundle?

Reply:
This is a README.md file, its just an example! Not sure where tree-shaking is supposed to work here?

Reply:
looks like a bad example